tests/test_streaming.py

```diff
@@ -779,7 +779,7 @@ async def ret_a(x: str) -> str:  # pragma: no cover
 class TestPartialOutput:
     """Tests for `ctx.partial_output` flag in output validators and output functions."""
 
-    # NOTE: When changing these tests:
+    # NOTE: When changing tests in this class:
     # 1. Follow the existing order
     # 2. Update tests in `tests/test_agent.py::TestPartialOutput` as well
```

Comment:
note to self that we should probably refactor these tests to be together instead of pointing between them using comments